### PR TITLE
Add support to pass parameters by axis

### LINKED ISSUE #268: Add support to pass parameters per axis
If one has e.g. a stack of microscopy images, rotation around one axis only might be desired.

The interface would be similar to e.g. `Crop`:

```python
import torchio as tio
apply_affine = tio.RandomAffine(degrees=(0, 0, 10))  # -10 to 10 around depth axis
apply_affine = tio.RandomAffine(degrees=(0, 0, 0, 0, -15, 10))  # -15 to 10 around depth axis
apply_affine = tio.RandomAffine(degrees=(0, 0, 0, 0, 10, 10))  # 10 around depth axis
```

### LINKED ISSUE #269: Inconsistent documentation in RandomAffine
```python
In [1]: import torchio as tio
tr = t
In [2]: tr = tio.RandomAffine(scales=1.2)

In [3]: tr.scales
Out[3]: (1.2, 1.2)
```

But the docs say:
> If only one value 𝑑 is provided, \s𝑖∼U(0,𝑑).

@GFabien this was introduced in #244. Do you know what's going on? Also, note that some wrong latex was introduced in that PR, as you can see in that example: the backslash before s in not necessary (https://torchio.readthedocs.io/transforms/augmentation.html#randomaffine). I've fixed it in some other places, but some others might still be around.

### PR DESCRIPTION
**Pull request by fepegar**
Fixes #268, fixes #269.

**Description**
Sometimes it is useful to be able to apply e.g. scaling or rotation around specific axes only. For example in videos / histology stacks, we might want rotation around the time / z axes only.

**Checklist**

- [x] I have read the [`CONTRIBUTING`](https://github.com/fepegar/torchio/blob/master/CONTRIBUTING.rst) docs and have a developer setup (especially important are `pre-commit`and `pytest`)
- [x] Non-breaking change (would not break existing functionality)
- [ ] Breaking change (would cause existing functionality to change)
- [x] Tests added or modified to cover the changes
- [x] Integration tests passed locally by running `pytest`
- [x] In-line docstrings updated
- [x] Documentation updated, tested running `make html` inside the `docs/` folder
- [x] This pull request is ready to be reviewed
- [x] If the PR is ready and there are multiple commits, I have [squashed them and force-pushed](https://www.w3docs.com/snippets/git/how-to-combine-multiple-commits-into-one-with-3-steps.html#force-pushing-commits-7)

### CODE REVIEW

**Comment by jxchen01** [comment]
Really appreciate this improvement!!! super helpful for microscopy image research. Looking forwarding to trying it out ~~
